__tests__/app.test.js

```diff
@@ -218,6 +218,89 @@ describe("POST /api/articles/:article_id/comments", () => {
+    return request(app)
+      .patch("/api/articles/4")
+      .send(incVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            title: "Student SUES Mitch!",
+            topic: "mitch",
+            author: "rogersop",
+            body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+            created_at: expect.any(String),
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            votes: 8,
+            article_id: 4,
+          })
+        );
+      });
+  });
+  test("200: responds with the updated article object with the vote property decremented when inc_votes is a negative integer", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send(decVotes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toEqual(
+          expect.objectContaining({
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: expect.any(String),
+            votes: 95,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            article_id: 1,
+          })
+        );
+      });
+  });
+  test("400: responds with an appropriate error message when request body does not contain the correct field", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ number: 9 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: Bad request");
+      });
+  });
+  test("400: responds with an appropriate error message when request body has a valid body field but the value of the field is invalid", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "String" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: Bad request");
+      });
+  });
+  test("400: responds with an appropriate error message when given an invalid article_id", () => {
+    return request(app)
+      .patch("/api/articles/notAnId")
+      .send(incVotes)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("400: Bad request");
+      });
+  });
+  test("404: responds with an appropriate error message when given a valid but non-existent article_id", () => {
+    return request(app)
+      .patch("/api/articles/99999")
+      .send(decVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("404: Not found");
+      });
+  });
+});
```

Comment:
superb extensive testing here, really well done I have no more suggested tests to add! 

You could extend your destructuring - so {body: {article} } for example would save you some dot notation!